frontend/src/metabase/parameters/components/ParameterLinkedFilters/ParameterLinkedFilters.unit.spec.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import userEvent from "@testing-library/user-event";
+import { renderWithProviders, screen } from "__support__/ui";
+import { UiParameter } from "metabase-lib/parameters/types";
+import { createMockUiParameter } from "metabase-lib/parameters/mock";
+import ParameterLinkedFilters from "./ParameterLinkedFilters";
+
+interface SetupOpts {
+  parameter: UiParameter;
+  otherParameters: UiParameter[];
+}
+
+const setup = ({ parameter, otherParameters }: SetupOpts) => {
+  const onChangeFilteringParameters = jest.fn();
+  const onShowAddParameterPopover = jest.fn();
+
+  renderWithProviders(
+    <ParameterLinkedFilters
+      parameter={parameter}
+      otherParameters={otherParameters}
+      onChangeFilteringParameters={onChangeFilteringParameters}
+      onShowAddParameterPopover={onShowAddParameterPopover}
+    />,
+  );
+
+  return { onChangeFilteringParameters, onShowAddParameterPopover };
+};
+
+describe("ParameterLinkedFilters", () => {
+  it("should toggle filtering parameters", () => {
+    const { onChangeFilteringParameters } = setup({
+      parameter: createMockUiParameter({
+        id: "p1",
+        name: "P1",
+      }),
+      otherParameters: [
+        createMockUiParameter({
+          id: "p2",
+          name: "P2",
+        }),
+      ],
+    });
+
+    userEvent.click(screen.getByRole("switch"));
```

Comment:
I think it'd be more useful if we could add the name to this switch to help understand better what toggle we're toggling. It's fine in this case since the markup doesn't contain any label for the toggle, however; it could be better for accessibility too. I think if we can make this queryable with a name then we will make a11y and dx better at the same time.